domain/src/main/java/be/xplore/githubmetrics/domain/domain/WorkflowRun.java

```diff
@@ -0,0 +1,47 @@
+package be.xplore.githubmetrics.domain.domain;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+public class WorkflowRun {
+    private final int id;
+    private final String name;
+    private RunStatus status;
+
+    public WorkflowRun(JsonNode json) {
+        this.id = json.get("id").asInt();
+        this.name = json.get("name").toString();
+        this.status = switch (json.get("status").asText()) {
```

Comment:
Keep you constructor clean. You could extract the status logic into a separate function